CIP-?/README.md

```diff
@@ -0,0 +1,99 @@
+
+2. **User Prompt:**
+
+   - If there is only one corresponding address resolved, the wallet could proceed with the only address for proceeding with the user transaction.
+   - If more than one corresponding address is detected, the wallet should display a prompt to the user, explaining the conflict, and providing options for resolution.
+
+3. **Resolution Options:**
+
+   - The prompt should offer the user the following options:
+
+     - Resolve with Project A
+     - Resolve with Project B
+     - etc...
+     - Cancel
+
+     > Detail UX implementation to be decided by the wallet as long as users are not confused with their choice, so only showing the project name is the minimal information needed, e.g. one for Cardano Name Service and another for adadomain
+
+4. **User Selection:**
+
+   - The user selects one of the options presented in the prompt. If they choose to resolve with Project A, the Project A's resolver is used; if they choose the other project, the respective resolver is used.
+
+5. **Default Project:**
+
+   - Users may have the option to set a default project for names with a colliding suffix in their wallet settings. This default project will be used for resolution unless changed by the user during the resolution prompt.
+   - Without user specification, there should always be an option prompting to user in case of domain name conflict.
+
+## Rationale: how does this CIP achieve its goals?
+
+This proposal embraces the decentralization property of a blockchain where it welcomes multiple domain name service that exists in the market without user experience competition at the infrastructure level. It ensures a user-friendly experience and allows users to make informed decisions in case of naming conflicts. It accommodates multiple projects with the same suffix in wallets while avoiding disputes and confusion, fostering the integration process of wallet and domain service.
+
+### Path to Active
+
+#### Acceptance Criteria
+
+- At least 3 of wallets listed below agree with the approach
+
+  - Begin <https://begin.is/>
+  - Eternl <https://eternl.io/>
+  - Flint <https://flint-wallet.com/>
+  - GeroWallet <https://www.gerowallet.io/>
+  - Lace <https://www.lace.io/>
+  - Nami <https://namiwallet.io/>
+  - NuFi <https://nu.fi/>
+  - RayWallet <https://raywallet.io/>
+  - Yoroi <https://yoroi-wallet.com/>
+
+  > Initial list of wallet adapted from CIP30, other wallet providers could to request to add to the list.
+
+#### Implementation Plan
+
+- Every participating Cardano domain service provider provides an address resolver SDK.
+- Every participating Cardano domain service provider provides either a desired prefix or suffix.
+- Wallet providers to execute and integrate with resolving address, domain service project to provide assistance.
+
+#### Participating Cardano Domain Service
```

Comment:
One way or another this item should be removed, since (bureaucratically) it's not a standard part of Path to Active:

- Based on the branch name I am assuming this is your homegrown solution: if so you can cite it as an implementation & it would address my original reservation (about not being specific enough) if you could talk in your Specification about how your own system works.  If considering it separately from the CIP, you might put it in Rationale to talk about how &^ why the CIP is designed to accommodate implementations like your own.
- Regardless of how much detail you go into, it should instead be an item in your Implementation Plan since you appear to have a working model (or plan to have one).

Reply:
So I just need to remove the line `#### Participating Cardano Domain Service` right? I think the content of the session is necessary to stay for community to take reference (and also its part of the implementation).

Reply:
Just please make sure that whatever content remains fits into the section where it appears to be.  If it's in either of the two Path to Active sections, it should fit that section as described here: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#path-to-active